api/http/handler/handler.go

```diff
@@ -231,6 +233,8 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		if h.OpenAMTHandler != nil {
 			http.StripPrefix("/api", h.OpenAMTHandler).ServeHTTP(w, r)
 		}
+	case strings.HasPrefix(r.URL.Path, "/api/fdo"):
+		http.StripPrefix("/api", h.FDOHandler).ServeHTTP(w, r)
```

Comment:
here we should check for a nil handler (same as with AMT)
```suggestion
                if h.FDOHandler != nil {
		      http.StripPrefix("/api", h.FDOHandler).ServeHTTP(w, r)
	        }
```